app/cms/doc-validation/test/webhook-content-check.server.test.ts

```diff
@@ -0,0 +1,135 @@
+import GetTreeResponse from "./fixtures/get_tree_response.json"
+import GetFileResponse from "./fixtures/get_file_response.json"
+
+// Name of the repo to be appended on URL
+const repoName = "fcl-js"
+
+// Id of check run to test. Can be found on the check run created payload.
+const checkRunId = "9925474144"
+
+// Id of the PR to validate. Can be found on the check run created payload.
+const prId = "1505"
+
+// Id of the git tree of the PR. Can be found on the check run created payload under pull_requests.head.sha
+const gitTreeId = "0d41854e6313b56bca34abe9e081df17a484ef5c"
+
+// URl-encoded file name to be downloaded
+const fileName = "docs%2Findex.md"
+
+// path to an output file to print patch requests. These are what Github receive from the App to display on the PR
+//const outputFile = "./patch_requests.txt"
+
+export const restHandlers = [
+  // Github endpoint to create a check run from a check suite requested call from Github
+  rest.post(
+    `https://api.github.com/repos/onflow/${repoName}/check-runs`,
+    async (req, res, ctx) => {
+      return res()
+    }
+  ),
+
+  // Github endpoint to update the check run on the PR
+  rest.patch(
+    `https://api.github.com/repos/onflow/${repoName}/check-runs/${checkRunId}`,
+    async (req, res, ctx) => {
+      // Pretty-print patch requests to an output file
+      // let reqBody = await req.json()
+      // reqBody = JSON.stringify(reqBody, null, 2)
+      // fs.appendFileSync("helloworld.txt", reqBody)
+
+      return res()
+    }
+  ),
+
+  // Github endpoint to get changed files from a PR
+  rest.get(
+    `https://api.github.com/repos/onflow/${repoName}/pulls/${prId}/files`,
+    (req, res, ctx) => {
+      return res(ctx.json(GetPrResponse))
+    }
+  ),
+
+  // Github endpoint to get the expanded git tree
+  rest.get(
+    `https://api.github.com/repos/onflow/${repoName}/git/trees/${gitTreeId}`,
+    (req, res, ctx) => {
+      return res(ctx.json(GetTreeResponse))
+    }
+  ),
+
+  // Github endpoint to download content of a file
+  rest.get(
+    `https://api.github.com/repos/onflow/${repoName}/contents/${fileName}`,
+    (req, res, ctx) => {
+      return res(ctx.json(GetFileResponse))
+    }
+  ),
+]
+
+// Setup msw server to mock requests
+const server = setupServer(...restHandlers)
+
+// Saves env to override them during testinig
+const env = process.env
+
+// Start server before all tests
+beforeAll(() => {
+  // Clean output file before tests
+  // fs.truncateSync("helloworld.txt", 0)
+
+  // Sets server to listen mode. 'error' does not allow any unmocked traffic pass through. 'bypass' does the opposite.
+  // server.listen({ onUnhandledRequest: "bypass" })
+  server.listen({ onUnhandledRequest: "error" })
+
+  // Environment variables for testing. GITHUB_APP_ID is a set value. BOT_GITHUB_TOKEN is your github personal access token
+  process.env.GITHUB_APP_ID = "230821"
+  process.env.BOT_GITHUB_TOKEN = "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"
```

Comment:
We should not commit secrets to github. Please rotate the token, and remove it from the code.

Reply:
token has been disabled. Issue has been addressed